### ISSUE TITLE
Secure Boot support

### ISSUE THREAD

**Issue by KSebion** [grml/grml#191]
**Is your software / feature request related to a problem? Please describe.**
I'm always frustrated when I have to disable secure boot to boot grml.

**Describe the solution you'd like**
Boot without disabling secure boot.

**Describe alternatives you've considered**
As an alternative solution, I used a combination of files from debian-live-11.1.0-amd64-standard.iso and grml64-small_2021.07.iso to boot grml with secure boot on.

**Disk usage of the software**
N/A

**The Grml version and flavour you checked that didn't have the feature you are asking for**

```shell
# grml-version
grml-small 2021.07 Release Codename JauKerl (2021-07-25)
```
I also tried grml64-full_2021.07.iso.

**Comment by mika** [comment]
Our build system (grml-live) already supports Secure Boot, but so far we didn't ship it with our public ISOs yet. We need to discuss how to handle this in the future.

**Comment by f466162** [comment]
As I just experienced the same, is there any progress on secure boot compatible ISOs? I'd highly appreciate this feature. :+1:

**Comment by adrianmsullivan** [comment]
I seriously doubt anyone who works on GRML knows or cares about this. We might as well pray to God for all the good that asking about it has done or will do.

**Comment by hvhaugwitz** [comment]
@mika Is there any way one can support you with this issue (e.g. by testing beta images)?

**Comment by mika** [comment]
> [@mika](https://github.com/mika) Is there any way one can support you with this issue (e.g. by testing beta images)?

Thanks for reaching out and the offer. It's not about development or testing, it's a business decision for us, as we provide Grml ISOs with Secure Boot support to commercial customers, e.g. via [Grml-Forensic](https://grml-forensic.org/).

**Comment by zeha** [comment]
@mika should we close this as not planned, or will anything change (soon)?

**Comment by mika** [comment]
> [@mika](https://github.com/mika) should we close this as not planned, or will anything change (soon)?

Ah good point. Maybe let's discuss this at our next development sprint?

**Comment by hvhaugwitz** [comment]
Please reconsider making such a basic feature as secure boot available to everyone, so that it is no longer required to toggle secure boot off an on again. Furthermore, secure boot does not appear to be advertised at all on the website for Grml-Forensic.

**Comment by dmaphy** [comment]
I agree with @hvhaugwitz, a basic feature like Secure Boot should be available for every end user, not just business customers.

**Comment by test-erik** [comment]
The current Grml 2025.12 does not boot with secure boot. Nowadays, this feature is not optional. Please provide an image, which boots with "secure boot" enabled.

**Comment by zeha** [comment]
SB is optional on platforms we currently consider supported.

Nevertheless, we have an ongoing discussion on the team what to do. At the end of the day, contrary to popular opinion, open source devs still need to be able to buy food. Let's see what comes out of it.
